tests/unit/conftest.py

```diff
@@ -86,3 +87,11 @@ def assert_cache_control(headers, cache_parts):
     assert (
         headers["Cache-Control"].split(", ") == Any.list.containing(cache_parts).only()
     )
+
+
+@pytest.fixture()
+def CheckmateClient():
+    with mock.patch(
+        "via.views.blocker.CheckmateClient", spec_set=True
+    ) as CheckmateClient:
+        yield CheckmateClient
```

Comment:
This should use our `patch` fixture, which'll handle the patching and unpatching and autospeccing etc:

```suggestion
@pytest.fixture()
def CheckmateClient(patch):
    return patch("via.views.blocker.CheckmateClient")
```

Reply:
This should be moved into `blocker_test.py` as well, since it's patching a name in `blocker.py`. I don't think tests for _other_ files (like `view_pdf.py`) should patch a name in `blocker.py` since that couples `view_pdf.py`'s tests to internal details of `blocker.py`.

`static_content_test.py` and `view_pdf_test.py` should use `httpretty` instead of this patch, see my other comments.

It's not quite the same thing as you're doing here, but we follow this pattern where `test_foo.py` should _only_ patch names in the corresponding `foo.py` module's namespace, we avoid patching names elsewhere or having global patch fixtures 'cos it can get really confusing. See this comment for why: https://github.com/hypothesis/via3/pull/222#discussion_r469865055